### PR TITLE
Add a /request metric with blame/result field.

### PR DESCRIPTION
**Pull request by yegle**
Part of https://github.com/google/exposure-notifications-verification-server/issues/773

The new metric will eventually replace all other metrics under /api/certificate.

### CODE REVIEW

**Review by icco — APPROVED** [review]
/lgtm

File: pkg/controller/certapi/certificate.go [hunk 1]
```diff
@@ -36,11 +37,18 @@ func (c *Controller) HandleCertificate() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		ctx := observability.WithBuildInfo(r.Context())
 
+		var blame = controller.BlameNone
+		var result = controller.APIResultOK()
+
+		defer stats.RecordWithTags(ctx, []tag.Mutator{blame, result}, mRequest.M(1))
```
Comment (yegle) [inline pkg/controller/certapi/certificate.go:43]: @icco Do you know if the linter (complain about blame/result variables not used later in the code) is correct?

I'm a bit fuzzy here, would this defer work as intended (use the blame/result variable's value right before return), or will it capture their current value and thus the blame/result variables later in the file is ignored?

Comment (icco) [reply pkg/controller/certapi/certificate.go:43]: I'm pretty sure it's the latter.

**Comment by yegle** [comment]
Can I get another LGTM?

File: pkg/controller/certapi/certificate.go (hunk 1 above)
Comment (yegle) [reply pkg/controller/certapi/certificate.go]: Fixed by capturing a pointer in the closure. Linter no longer complain about unused variables.

**Review by icco — APPROVED** [review]
/lgtm

**Pull request merged**